src/services/service.js

```diff
@@ -32,14 +34,29 @@ class Service {
     this.deployments = new Deployments(this.client)
     this.routing = new Routes(this.client)
     this.letsencrypt = new LetsEncrypt(this.client)
+
     const token = localStorage.getItem("token")
-    if(token) this.client.setToken(token);
+    if (token) this.client.setToken(token);
   }
 
   setToken(token) {
     this.client.setToken(token)
   }
 
+  refreshToken(token) {
+    return new Promise((resolve, reject) => {
+      this.client.getJSON("/v1/config/refresh-token", { headers: { "Authorization": token } }).then(({ status, data }) => {
```

Comment:
This is not how we set token. Token is prefixed with `Bearer `. So the value of the Authorization header should be: `Bearer ${token}`